x/programs/rust/wasmlanche_sdk/src/host/balance.rs

```diff
@@ -0,0 +1,18 @@
+// The balance module exposes helpers for transferring tokens between accounts.
+#[link(wasm_import_module = "balance")]
+extern "C" {
+    #[link_name = "get"]
+    fn _get(
+        caller_id: i64,
+        key_ptr: *const u8,
+        key_len: usize,
+        value_ptr: *const u8,
+        value_len: usize,
+    ) -> i32;
+
+    #[link_name = "get_bytes_len"]
+    fn _get_bytes_len(caller_id: i64, key_ptr: *const u8, key_len: usize) -> i32;
+
+    #[link_name = "get_bytes"]
+    fn _get_bytes(caller_id: i64, key_ptr: *const u8, key_len: usize, val_len: i32) -> i32;
+}
```

Comment:
Which IDE are you using? You should make it so the IDE automatically adds a new line at the end of the file. If you're using Vscode, I can find you the stack-overflow post on how to do it.

Reply:
hmm I thought `cargo fmt` did that or I guess why doesn't it? anyways I will look into it.

Reply:
I thought cargo fmt did that too... Weird